src/coreclr/nativeaot/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/ModuleList.cs

```diff
@@ -349,7 +349,7 @@ public struct NativeFormatModuleInfoEnumerator
         /// <summary>
         /// Current _modules element that should be returned by Current (updated in MoveNext).
         /// </summary>
-        private NativeFormatModuleInfo _currentModule;
+        private NativeFormatModuleInfo? _currentModule;
```

Comment:
I think we can undo the *.cs changes under System.Private.TypeLoader since nullable is now disabled.